doc/new.md

```diff
@@ -4,9 +4,13 @@ The **new** command of the [Winget-Create](../README.md) tool begins the process
 
 ## Usage
 
-`WingetCreateCLI.exe new [<url>] [\<options>]`
+`WingetCreateCLI.exe new [<urls>] [\<options>]`
 
-The **new** command can be called with the optional URL.  If the URL is provided, **Winget-Create** will download the installer as it begins.  If the URL is not included, the user will need to add it when prompted.
+The **new** command can be called with the optional URL(s). **Winget-Create** will download each provided installer as it begins.  If the URL is not included, the user will need to add it when prompted. When prompted, use a comma to separate each installer url you wish to include in your generated manifest.
```

Comment:
It's unfortunate that when multiple urls are provided on the command line, they're space delimited, which is different than described below. Can you incorporate that into the example on line 7?

Reply:
Added an extra example for the command line scenario.